Configuration/Settings.yaml

```diff
@@ -6,20 +6,14 @@ Flowpack:
     # the maximum public cache control header sent
     # set to 0 if you do not want to send public CacheControl headers
     maxPublicCacheTime: 86400
+
 Neos:
   Flow:
     http:
-      chain:
-        'process':
-          chain:
-            requestInterceptor:
-              position: 'start 999999'
-              component: 'Flowpack\FullPageCache\Http\RequestInterceptorComponent'
-            addCacheHeader:
-              position: 'after setHeader'
-              component: 'Flowpack\FullPageCache\Http\CacheControlHeaderComponent'
-        'postprocess':
-          chain:
-            requestStorage:
-              position: 'after standardsCompliance'
-              component: 'Flowpack\FullPageCache\Http\RequestStorageComponent'
+      middlewares:
+        'fullPageCache_RequestCache':
+          middleware: 'Flowpack\FullPageCache\Middleware\RequestCacheMiddleware'
+          position: 'after session 20'
```

Comment:
Note: The middleware that stores the response, needs to come "before" session, so that session MW can set the `Set-Cookie` header and the caching middleware can avoid caching that response

Reply:
Actually that is why this currently is behind "session" since currently the flow session is asked for deciding about cachability. I already wondered whether it would be better to just look at the cookies of the request and put this before "session".

Reply:
This is now adjusted and the request cache middleware does only look at the incoming cookies and arguments to decide wether a request is cacheable and thus this is now placed after trustedProxies now as @albe suggested.